Altis_Life.Altis/core/functions/fn_startLoadout.sqf

```diff
@@ -0,0 +1,81 @@
+#include "..\..\script_macros.hpp"
+/*
+    File: fn_startLoadout.sqf
+    Author: Casperento
+
+    Description:
+    Loads a custom loadout on player when he got a new life
+*/
+private _pUniform = M_CONFIG(getArray,"Loadouts",str(playerSide),"uniform");
+private _pVest = M_CONFIG(getArray,"Loadouts",str(playerSide),"vest");
+private _pBackpack = M_CONFIG(getArray,"Loadouts",str(playerSide),"backpack");
+private _pWeapon = M_CONFIG(getArray,"Loadouts",str(playerSide),"weapon");
+private _pMagazines = M_CONFIG(getArray,"Loadouts",str(playerSide),"mags");
+private _pItems = M_CONFIG(getArray,"Loadouts",str(playerSide),"items");
+private _linkedItems = M_CONFIG(getArray,"Loadouts",str(playerSide),"linkedItems");
+
+if !(_pUniform isEqualTo []) then {
+    if (playerSide isEqualTo civilian) then {
+        _pUniform = selectRandom _pUniform;
+        if (!(_pUniform isEqualTo []) && {!((_pUniform select 0) isEqualTo "") && {([(_pUniform select 1)] call life_fnc_levelCheck)}}) then {
+            player addUniform (_pUniform select 0);
+        };
+    } else {
+        _pUniform apply {
+            if (!(_x isEqualTo []) && {!((_x select 0) isEqualTo "") && {([(_x select 1)] call life_fnc_levelCheck)}}) then {
+                player addUniform (_x select 0);
+            };
+        };
+    };
+};
+
+if !(_pVest isEqualTo []) then {
+    _pVest apply {
+        if (!(_x isEqualTo []) && {!((_x select 0) isEqualTo "") && {([(_x select 1)] call life_fnc_levelCheck)}}) then {
+            player addVest (_x select 0);
+        };
+    };
+};
+
+if !(_pBackpack isEqualTo []) then {
+    _pBackpack apply {
+        if (!(_x isEqualTo []) && {!((_x select 0) isEqualTo "") && {([(_x select 1)] call life_fnc_levelCheck)}}) then {
+            player addBackpack (_x select 0);
+        };
+    };
+};
+
+if !(_pWeapon isEqualTo []) then {
+    _pWeapon apply {
+        if (!(_x isEqualTo []) && {!((_x select 0) isEqualTo "") && {([(_x select 1)] call life_fnc_levelCheck)}}) then {
+            player addWeapon (_x select 0);
+        };
+    };
+};
+
+if !(_pMagazines isEqualTo []) then {
+    _pMagazines apply {
+        if (!(_x isEqualTo []) && {!((_x select 0) isEqualTo "") && (_x select 1) > 0 && {([(_x select 2)] call life_fnc_levelCheck)}}) then {
```

Comment:
Any reason why `(_x select 1) > 0` isn't lazy evaluated?

Reply:
The end user